### ISSUE TITLE
Visual layout issues of Techniques for WCAG 2.0

### ISSUE THREAD

**Issue by takenspc** [w3c/wcag#172]
In https://www.w3.org/TR/WCAG-TECHS/general.html and other pages, table of contents overlaps main contents. I cannot read the main contents behind table of contents. Please fix this issue.

![Screenshot of General Techniques for WCAG 2.0 (table of contents overlaps main contents)](https://cloud.githubusercontent.com/assets/270905/14194196/bb6d8f5e-f7e6-11e5-98f9-7f71b3dde98e.png)

I found that following pages have this issue.

https://www.w3.org/TR/WCAG-TECHS/general.html
https://www.w3.org/TR/WCAG-TECHS/html.html
https://www.w3.org/TR/WCAG-TECHS/css.html
https://www.w3.org/TR/WCAG-TECHS/client-side-script.html
https://www.w3.org/TR/WCAG-TECHS/server-side-script.html
https://www.w3.org/TR/WCAG-TECHS/smil.html
https://www.w3.org/TR/WCAG-TECHS/text.html
https://www.w3.org/TR/WCAG-TECHS/aria.html
https://www.w3.org/TR/WCAG-TECHS/flash.html
https://www.w3.org/TR/WCAG-TECHS/silverlight.html
https://www.w3.org/TR/WCAG-TECHS/pdf.html
https://www.w3.org/TR/WCAG-TECHS/failures.html

**Comment by awkawk** [comment]
We are aware of the issue and are working to address it.

**Comment by awkawk** [comment]
Fixed.
